### ISSUE TITLE
[Accessibility][High Contrast] ToolStrip grip icon colour contrast is less than 4.5:1 in Normal Mode and HC Desert mode

### ISSUE THREAD

**Issue by LeafShi1** [dotnet/winforms#12663]
## Minimal Reproduction Scenario
1. Create a WinForms Project.
2. Add a toolStrip to the project.
3. Test the text grip icon in toolStrip by using Accessibility Insight tools.

## Current Behavior
The colour contrast is less than 4.5:1 in Normal Mode and HC Desert Mode

**Normal mode**
![Image](https://github.com/user-attachments/assets/ba316b28-5c31-4058-944b-673b804fc02c)

**HC Aquatic**
![Image](https://github.com/user-attachments/assets/eb1e2e15-2908-4fe3-b623-3b488c46ef3d)

**HC Desert** 
![Image](https://github.com/user-attachments/assets/e4c33163-249a-48e0-866a-64d656d532e8)

**HC Dusk**
![Image](https://github.com/user-attachments/assets/f9706f23-12f1-4c7e-b57c-2f08d5fbdc94)

**HC Night Sky**
![Image](https://github.com/user-attachments/assets/0d3fe566-9b73-489e-b1c7-f1942812429d)

## Expected Behavior
The colour contrast is more than 4.5:1 in Normal Mode and all HC Mode

## Test Environment
Microsoft Visual Studio Enterprise 2022 (64-bit) - main
Version 17.13.0 Preview 3.0 [35604.161.main]

## Have you experienced this same issue with .NET Framework designer?
Yes

## Have you experienced this same issue with previous previews of VisualStudio?
Yes, reproduce on VS 17.12

## More Info
- This issue also reproduces on MenuStrip
- This issue also reproduces in runtime Dark Mode
![Image](https://github.com/user-attachments/assets/79001419-3c5e-487e-b4b4-9729e478c72d)

**Comment by LeafShi1** [comment]
The original issue [#6208](https://github.com/microsoft/winforms-designer/issues/6208)

**Comment by Tanya-Solyanik** [comment]
@Epica3055  - if this issue is easily fixable by replacing the colors, please go ahead with the fix, if not, we'll wait for a customer report.

**Comment by Zheng-Li01** [comment]
Verified the issue on latest .NET SDK 10.0.100-alpha.1.25058.38 build that the issue has been fixed as below screenshot.

Only the issue still reproduces when setting the RenderMode as "System" on Normal mode. after confirmed that the current behavior is expected due to cannot do any changes with the **System** mode for now.

 **Win11 Results:**
![Image](https://github.com/user-attachments/assets/35f41fbb-b3bb-4a3e-8a75-27a56ba713b9)

**Win10 Results:**
![Image](https://github.com/user-attachments/assets/7037a8f6-50e0-4105-a5f5-4be48bf49328)
